CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.6.2]
+
+### Added
+* Added documentation to [InSAR Product Guide](docs/guides/insar_product_guide.md#adaptive-phase-filter) for the adaptive phase filter parameter option
+
```

Comment:
Do we also want to call out the extra re-structuring that you did?

Reply:
I suppose that's fair.

Reply:
Addressed with [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/379/commits/0b137e5e37ddbd2129a104f59bfee1261c2c1167)